src/basics/arrays/min-max-average/index.js

```diff
@@ -4,6 +4,16 @@
  * stats([1, 2, -2, 3]) //=> {min: -2, max: 3, average: 1}
  * Bonus: try to implement this without a `for` or `while` loop
  */
-function stats(numbers) {}
+function stats(numbers) {
+    numbers.sort();
+    var min = numbers[0];
+    var max = numbers[numbers.length-1];
+    var sum = numbers.reduce(function (a,b) {
+        return a+b ;
+    },0);
+    var average = sum/numbers.length ;
+    return {'min' : min , 'max' : max , 'average': average} 
+
+}
```

Comment:
if we define `N` to be the length of numbers array, then what is the complexity of this function?
Can make it linear (`O(N)`)?

Reply:
If we define N to be the length of the array, I think the complexity would be nlog(n) 
I think we can make it linear by implementing the sort method